### PR TITLE
chore: docker-compose postgres healthcheck

### PR DESCRIPTION
**Pull request by patriknw**
* instead of sleep in CI

Thanks @pvlugter

### CODE REVIEW

**Review by johanandren — APPROVED** [review]
Neat!

**Comment by johanandren** [comment]
But:
> No such container: docker_postgres-db_1

**Review by octonato — APPROVED** [review]

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -67,10 +67,8 @@ jobs:
 
       - name: Start DB
         run: |-
-          docker-compose -f docker/docker-compose-postgres.yml up -d
-          # TODO: could we poll the port instead of sleep?
-          sleep 10
-          docker exec -i docker_postgres-db_1 psql -U postgres -t < ddl-scripts/create_tables_postgres.sql
+          docker-compose -f docker/docker-compose-postgres.yml up --build --detach --wait
```
Comment (pvlugter) [inline .github/workflows/build-test.yml:70]: Don't need `--build` — only when it's building a custom image from a dockerfile. So just need the `--wait`.

**Comment by pvlugter** [comment]
I'll take a look at this.

**Comment by pvlugter** [comment]
Checked some things in an empty github action. Note that there is no ubuntu 23.04 available, 22.04 is the latest runner still. Looks like there are two versions of docker-compose in the [image](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md), both v1 and v2. Where `docker-compose` is v1, while using `docker compose` will use the v2 plugin. Trying fresh in #437, where this is working now:

```
 Container postgres-db  Creating
 Container postgres-db  Created
 Container postgres-db  Starting
 Container postgres-db  Started
 Container postgres-db  Waiting
 Container postgres-db  Healthy
```

**Comment by pvlugter** [comment]
Will close this, replaced by #437. And will update https://github.com/akka/akka-projection/pull/890 in place.

**Pull request closed without merging**